packages/aws-cdk/test/_helpers/jest-bufferedconsole.ts

```diff
@@ -99,7 +99,7 @@ export default class TestEnvironment extends NodeEnvironment implements JestEnvi
 type TestDescription = PartialBy<Pick<Circus.TestEntry, 'name' | 'parent'>, 'parent'>;
 
 // Utility type to make specific fields optional
-type PartialBy<T, K extends keyof T> = Omit<T, K> & Partial<Pick<T, K>>
+type PartialBy<T, K extends keyof T> = Omit<T, K> & Partial<Pick<T, K>>;
```

Comment:
This wasn't picked up before because the default eslint `semi` rules doesn't do TypeScript.